presentation/src/main/java/com/thanht/foodyentrytask/util/Mapper.kt

```diff
@@ -0,0 +1,8 @@
+package com.thanht.foodyentrytask.util
+
+import com.thanht.domain.model.CityModel
+import com.thanht.foodyentrytask.home.list.CityInfo
+
+fun CityModel.toCityInfo(): CityInfo = CityInfo(id, name)
+
+fun List<CityModel>.toCityInfoList(): List<CityInfo> = map { it.toCityInfo() }
```

Comment:
hmm, if it's simple transform use map { CityInfo(it.id, it.name) } no need declare mapper for it, read inside `iosched` it used to map function for simple mapping.